test/dapis/DapiServer.sol.js

```diff
@@ -2918,11 +2918,23 @@ describe('DapiServer', function () {
         });
       });
       context('Data feed ID is zero', function () {
-        it('reverts', async function () {
-          const dapiName = hre.ethers.utils.formatBytes32String('My dAPI');
-          await expect(
-            dapiServer.connect(roles.dapiNameSetter).setDapiName(dapiName, hre.ethers.constants.HashZero)
-          ).to.be.revertedWith('Data feed ID zero');
+        context('Sender is dAPI name setter', function () {
+          it('sets dAPI name', async function () {
+            const dapiName = hre.ethers.utils.formatBytes32String('My dAPI');
+            await dapiServer.connect(roles.dapiNameSetter).setDapiName(dapiName, beaconSetId);
+            await expect(dapiServer.connect(roles.dapiNameSetter).setDapiName(dapiName, hre.ethers.constants.HashZero))
```

Comment:
Maybe here in this same test you could also add another call to `setDapiName` passing some random hash that is not the `HashZero` just to make sure happy path is working in all cases.